### PR TITLE
request: fix request made by a librarian

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Changes messages.
* Improves button appearance.
* Disables 'delete' button when a request is done.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

## Why are you opening this PR?

To implement task 1353 of US657
https://tree.taiga.io/project/rero21-reroils/task/1353?kanban-status=1224894
See https://gitter.im/rero/reroils-PO?at=5e65f10102dbe022a175fa76

## How to test?

1. Go to the detailed view of a document of your library.
0. Buttons' design should be uniformized with other buttons of the app.
0. Messages should match PO specifications (see https://gitter.im/rero/reroils-PO?at=5e65f10102dbe022a175fa76 or https://tree.taiga.io/project/rero21-reroils/task/1353?kanban-status=1224894)
2. Make a request for a patron.
3. When it's done, the delete button should be dark and delete should be forbidden.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/document-detail-view/holding-item/holding-item.component.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { BsModalService } from 'ngx-bootstrap';
 import { RecordPermission, RecordPermissionService } from '../../../../service/record-permission.service';
 import { UserService } from '../../../../service/user.service';
 import { ItemRequestComponent } from '../item-request/item-request.component';
+import { first } from 'rxjs/operators';
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding-item/holding-item.component.ts]: sort imports

**Comment by AoNoOokami** [comment]
> Item detailed page
> 
>     * you decided to implement the function later? I don't care, but then for sure we will have to come back on it.
> 
> 
> Button
> 
>     * Icons have been replaced by labels. Not sure it is the right thing to do (I just wanted more _spaced_ buttons), as we have other cases with actions on a list of resources that still use icons. Anyway, it is ok for now.
> 
>     * Button "Delete" is still highlighted. We should use the Bootstrap outline buttons, because this is a secondary action. Color is good!
> 
>     * Button "Request" must not be highlighted (primary), it is not a call to action. This is the case for the end user, not for the librarian.
> 
> 
> The feature works. I think in that state, mainly the "delete" button is very confusing. We could also think about an issue where we correct all buttons according to our charter.

I'll check for the item detailed page.

For the button, I thought I was following the UXChart, so perhaps there are some things to clarify about it.

**Comment by pronguen** [comment]
> For the button, I thought I was following the UXChart, so perhaps there are some things to clarify about it.

@AoNoOokami There should be in principle 1 or 0 primary action per page. This action is: Save (when an editor is open), Delete (when a popup ask to confirm a deletion), for the patron view eventually Request (on document detailed view, to encourage them to request books).

**Comment by iGormilhit** [comment]
@pronguen So for this page, the primary is the edit button on the right top. Delete and remove are to be red (danger), and all the others are secondary button. Rights ?

**Comment by pronguen** [comment]
@iGormilhit I think we must not have a primary action for all pages... in that case, no primary action is ok according to me (you don't go on the document detail view always in order to *edit* it).
In any case, we must avoid to have more than one primary action in a series a buttons next to each others!

**Pull request merged**